Dockerfile

```diff
@@ -96,8 +96,20 @@ RUN set -eux ; \
     mv dumb-init /usr/local/bin/dumb-init
 
 #byteman test for development
-RUN curl -Lo /opt/byteman.jar https://repo.maven.apache.org/maven2/org/jboss/byteman/byteman/4.0.23/byteman-4.0.23.jar \
-    && chmod o+r /opt/byteman.jar
+ARG BYTEMAN_VERSION=4.0.25
+ARG BYTEMAN_HOME=/opt/byteman/
+RUN sudo yum install unzip -y && \
```

Comment:
Please add `unzip` package in existing install command:

https://github.com/apache/ozone-docker-runner/blob/d2fd9a2d59126546a521fa52a739c918eb125ee8/Dockerfile#L53-L54